L1Trigger/Phase2DAQProducer/plugins/Phase2DAQProducer.cc

```diff
@@ -0,0 +1,218 @@
+#include <memory>
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/one/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+#include "DataFormats/TrackReco/interface/Track.h"
```

Comment:
Why does a cluster to raw converter need include files for tracks? Please try to minimise the number of include files you need. Code should be as simple as possible.

Reply:
yup, removed. Again, some old test.